task01/index.js

```diff
@@ -0,0 +1,7 @@
+const times = 5;
+const callback = count => console.log(count);
+const loop = (times = 0, callback = null) => {    
+    (typeof callback === 'function')?[...Array(times)].map((value,index) => callback(index+1)):console.log('Callback is undefined!');
```

Comment:
```suggestion
    (typeof callback === 'function')?[...Array(times)].forEach((value,index) => callback(index+1)):console.log('Callback is undefined!');
```
map() дополнительно создает новый массив,  а нам надо просто по элементам пройтись:)

Reply:
ок